package.json

```diff
@@ -28,75 +28,75 @@
     "prepare": "husky install"
   },
   "dependencies": {
-    "@angular/animations": "13.0.2",
-    "@angular/cdk": "13.0.2",
-    "@angular/common": "13.0.2",
-    "@angular/compiler": "13.0.2",
-    "@angular/core": "13.0.2",
-    "@angular/forms": "13.0.2",
-    "@angular/material": "13.0.2",
-    "@angular/platform-browser": "13.0.2",
-    "@angular/platform-browser-dynamic": "13.0.2",
-    "@angular/router": "13.0.2",
-    "@ngrx/store": "13.0.1",
-    "@nrwl/angular": "13.2.2",
-    "@nrwl/nx-cloud": "12.5.4",
+    "@angular/animations": "13.1.1",
+    "@angular/cdk": "13.1.1",
+    "@angular/common": "13.1.1",
+    "@angular/compiler": "13.1.1",
+    "@angular/core": "13.1.1",
+    "@angular/material": "13.1.1",
+    "@angular/platform-browser": "13.1.1",
+    "@angular/platform-browser-dynamic": "13.1.1",
+    "@angular/router": "13.1.1",
+    "@ngrx/store": "13.0.2",
+    "@nrwl/angular": "13.4.3",
+    "@nrwl/nx-cloud": "13.0.2",
     "@testing-library/dom": "^8.11.1",
-    "@testing-library/user-event": "^13.5.0",
-    "rxjs": "^7.4.0",
+    "rxjs": "^7.5.1",
     "tslib": "~2.3.1",
     "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "13.0.3",
-    "@angular-eslint/eslint-plugin": "12.6.1",
-    "@angular-eslint/eslint-plugin-template": "12.6.1",
-    "@angular-eslint/template-parser": "12.6.1",
-    "@angular/cli": "13.0.3",
-    "@angular/compiler-cli": "13.0.2",
-    "@angular/language-service": "13.0.2",
-    "@nrwl/cli": "13.2.2",
-    "@nrwl/eslint-plugin-nx": "13.2.2",
-    "@nrwl/jest": "13.2.2",
-    "@nrwl/linter": "13.2.2",
-    "@nrwl/node": "13.2.2",
-    "@nrwl/nx-plugin": "13.2.2",
-    "@nrwl/workspace": "13.2.2",
+    "@angular-devkit/build-angular": "13.1.2",
+    "@angular-eslint/eslint-plugin": "13.0.1",
+    "@angular-eslint/eslint-plugin-template": "13.0.1",
+    "@angular-eslint/template-parser": "13.0.1",
+    "@angular/cli": "13.1.2",
+    "@angular/compiler-cli": "13.1.1",
+    "@angular/forms": "13.1.1",
```

Comment:
@timdeschryver 
The 'fix' / prevent `@angular/forms` and `@testing-library/user-event` being exported in the generated `package.json` I moved them to the `devDependencies`, since they are only used in 'dev' -> tests

don't think it is fixed in NX yet